src/diffusers/models/vae.py

```diff
@@ -280,6 +280,292 @@ def custom_forward(*inputs):
         return sample
 
 
+class UpSample(nn.Module):
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        activate_before: str = "none",
+        activate_after: str = "none",
+        upsample_type: str = "deconv",
```

Comment:
```suggestion
        activate_before: str = "none",
        activate_after: str = "none",
        upsample_type: str = "deconv",
```

Do we need all these options ? To begin with could we maybe only add the options that are need for inference in a first time?

Reply:
In those checkpoints:
- https://huggingface.co/cross-attention/asymmetric-autoencoder-kl-x-2/blob/main/config.json
- https://huggingface.co/cross-attention/asymmetric-autoencoder-kl-x-1-5/blob/main/config.json

it looks like only the default values are used, could we for now just remove all other options than the ones used for inference?